instructions/docs/add-the-encryption-sdk.md

```diff
@@ -28,16 +28,16 @@ Now you will add the AWS Encryption SDK to encrypt close to where the data origi
 
 Make sure you are in the `exercises` directory for the language of your choice:
 
-```Java
-~/environment/workshop/exercises/java
+```bash tab="Java"
+cd ~/environment/workshop/exercises/java
 ```
 
-```Javascript Node.js
-~/environment/workshop/exercises/node-javascript
+```bash tab="JavaScript Node.JS"
+cd ~/environment/workshop/exercises/node-javascript
 ```
 
-```Python
-~/environment/workshop/exercises/python
+```bash tab="Python"
+cd ~/environment/workshop/exercises/python
 ```
 
 `cd` into the `add-esdk-start` directory.
```

Comment:
Why not just do it all in one step? Make sure you are in the add-esdk-start exercises directory for your chosen language:
cd ~/environment/workshop/exercises/java/add-edsk-start"